src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+import { PeopleContext } from '../../peopleContext';
+import { PeopleFilters } from '../PeopleFilters';
+import { useSearchParams } from 'react-router-dom';
+import { filterPeople } from '../../utils/filterPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [filteredPeople, setFilteredPeople] = useState<Person[]>(people);
```

Comment:
You've initialized `filteredPeople` with `people` which is an empty array initially. This is unnecessary because `filteredPeople` will be updated in the useEffect hook when `people` changes. So, you can just initialize `filteredPeople` as an empty array.